src/common/logging.cpp

```diff
@@ -89,18 +93,23 @@ void otbrLogInit(const char *aProgramName, otbrLogLevel aLevel, bool aPrintStder
 {
     const char *ident;
 
-    assert(aProgramName != nullptr);
     assert(aLevel >= OTBR_LOG_EMERG && aLevel <= OTBR_LOG_DEBUG);
+    otbrLogSyslogSetEnabled(!aSyslogDisable);
```

Comment:
What's the purpose of moving this line?

Reply:
This function is a common function when using Android log system or linux log system. So I moved it to the beginning of this method.